### PR TITLE
mobile nav

### PR DESCRIPTION
**Pull request by JamesKemp296**

### CODE REVIEW

**Comment by JamesKemp296** [comment]
Screenshot from 2020-03-18 14-32-37
Screenshot from 2020-03-18 14-32-46

File: apps/webClient/src/components/MobileNav/index.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import Drawer from "@material-ui/core/Drawer";
+import Button from "@material-ui/core/Button";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import MenuIcon from "@material-ui/icons/Menu";
+import CloseIcon from "@material-ui/icons/Close";
+
+interface Props {
+  children: ReactNode;
```
Comment (cbaron) [inline apps/webClient/src/components/MobileNav/index.tsx]: `Props` should be empty as we don't use any props If I read this correctly.

File: apps/webClient/src/components/View/index.tsx [hunk 2]
```diff
@@ -2,15 +2,16 @@ import React from "react";
...
 import { makeStyles } from "@material-ui/styles";
 import Container from "@material-ui/core/Container";
 import Box from "@material-ui/core/Box";
 import Grid from "@material-ui/core/Grid";
 import Logo from "../Logo";
+import { smallScreenMaxBreakpoint } from "../../utils";
+import MobileNav from "../../components/MobileNav";
 
-interface Props {
-  children: React.ReactNode;
```
Comment (cbaron) [inline apps/webClient/src/components/View/index.tsx:12]: Not sure what the confusion is here - we _do_ use `children` in this component.

**Review by cbaron — CHANGES_REQUESTED** [review]
Very nice.  Just a concern on the component prop interface declarations

**Review by cbaron — APPROVED** [review]

**Pull request merged**